### ISSUE TITLE
Spike in your Stats Push Notification takes you to the wrong blog stats

### ISSUE THREAD

**Issue by daniloercoli** [wordpress-mobile/WordPress-Android#9741]
I've received a `Spike in your Stats` push notification for a personal hosted on wpcom, but  tapping on the link at the bottom of the notification details the app opened the Stats screen and loaded stats for another blog (I guess it was the blog already selected in the app, or the default blog of my account, that's a test blog).

![photo5922593093700399179](https://user-images.githubusercontent.com/518232/56885310-74cb3f00-6a6c-11e9-8363-0691c2bf1934.jpg)

When tapped on the link the app opened the Stats screen about a test blog of mine:

![photo5922593093700399178](https://user-images.githubusercontent.com/518232/56885307-7268e500-6a6c-11e9-9d5d-7bde0e566f7c.jpg)

cc @planarvoid

**Comment by ravishanker** [comment]
This is probably outdated.  This notification doesn't even seem to exist anymore.  Hence closing

**Comment by guarani** [comment]
👋 Thanks Ravi. I closed this as not planned.

**Comment by guarani** [comment]
@irfano, did you have a chance to verify this issue? I see it was moved from the Backlog to Ready.

I'd thought of the Backlog column as a place to collect issues before verification and the Ready column as a place to put verified issues. 

<img src="https://github.com/wordpress-mobile/WordPress-Android/assets/1898325/b014a12f-2115-4d8f-a65d-c764964a8529" width="500" />

Happy to discuss and settle on a workflow that suits us.

**Comment by guarani** [comment]
@irfano, did you have a chance to verify this issue? I see it was moved from the Backlog to Ready.

I'd thought of the Backlog column as a place to collect issues before verification and the Ready column as a place to put verified issues. 

<img src="https://github.com/wordpress-mobile/WordPress-Android/assets/1898325/b014a12f-2115-4d8f-a65d-c764964a8529" width="500" />

Happy to discuss and settle on a workflow that suits us.

**Comment by irfano** [comment]
@guarani, I didn't reproduce all issues in the "Ready" column. I investigated them, attempted to reproduce them, and moved them to the "Ready" column even if couldn't reproduce them, but I believed they were likely active issues based on my intuition.

Regarding this specific issue, I remember that I found the notification string, "...appears to be getting more traffic..." oninour source code. However, I've now noticed that the string does not exist on the code, there are only old discussions related to this notification. So, Ravi's assessment was correct. Good to close this. Thanks, @ravishanker!
